rstudio/rhel9-python-3.11/Dockerfile.cpu

```diff
@@ -169,7 +173,7 @@ COPY ${RSTUDIO_SOURCE_CODE}/utils utils/
 COPY ${RSTUDIO_SOURCE_CODE}/run-rstudio.sh ${RSTUDIO_SOURCE_CODE}/setup_rstudio.py ${RSTUDIO_SOURCE_CODE}/rsession.sh ${RSTUDIO_SOURCE_CODE}/run-nginx.sh ./
 
 # Unregister the system
-RUN subscription-manager remove --all && subscription-manager unregister && subscription-manager clean
+RUN [ ! -f '/var/tmp/.subscription-initially-not-present' ] || (subscription-manager remove --all && subscription-manager unregister && subscription-manager clean)
```

Comment:
Can you, please, enlighten me in which scenario we do have the subscription present which leads us not to remove it from the build machine? :thinking: you're thinking about the local build somehow?

Reply:
the github actions build added in this pr, it is subscribed by mounting the certs into the container, and therefore subscription-manager unregister should not be called at the end, because it would destroy the creds that are stored as repo secret

Reply:
I see. One thing - maybe it would be slightly easier to understand if there isn't that many negations? E.g., let's create a file with the name `subscription-present-initially`. And remove the `!` in the first condition?

Just a proposal, I don't insist. It seems slightly more readable to me.

Reply:
okay, I can see now, that this part of the code is outdated and you changed this already. Thanks!

Reply:
yes, I needed a negative so that i could use `[ ] ||`

later decided that it's worth redoing this